vulnerability-age-data.py

```diff
@@ -27,12 +26,10 @@
 import logging
 import sys
 
+from cyhy.db import database
```

Comment:
If we're switching to `cyhy` for database functionality then I feel like the raw db access on lines 64, 71, and 90 should switch to the appropriate Doc classes to keep in line with other projects using that library.

Reply:
Addressed in 00a7d7b.